charts/tidb-lightning/values.yaml

```diff
@@ -59,6 +59,19 @@ targetTidbCluster:
   #secretUserKey: user
   #secretPwdKey: password
 
+# Whether enable the TLS connection between TiDB cluster components.
+# If enabled, a Secret named "<targetTidbCluster.name>-lightning-cluster-secret" must exist.
+# To create this Secret: kubectl create secret generic <targetTidbCluster.name>-lightning-cluster-secret --namespace=<namespace> --from-file=tls.crt=<path/to/tls.crt> --from-file=tls.key=<path/to/tls.key> --from-file=ca.crt=<path/to/ca.crt>
+# tlsCluster:
+#   enabled: true
+
+# Whether enable the TLS connection with the TiDB MySQL protocol port.
+# if enabled, a Secret named `secretName` (if specified) or `${targetTidbCluster.name}-tidb-client-secret` must exist.
+# To create this Secret: kubectl create secret generic ${targetTidbCluster.name}-tidb-client-secret --namespace=<namespace> --from-file=tls.crt=<path/to/tls.crt> --from-file=tls.key=<path/to/tls.key> --from-file=ca.crt=<path/to/ca.crt>
+# tlsClient:
+#   enabled: true
+#   secretName: ${targetTidbCluster.name}-tidb-client-secret
```

Comment:
Should we need to rename it to `tlsClientSecretName` and move it out of `tlsClient`?

Reply:
@csuzhangxc I think we should rename it to `tlsClientSecretName` to keep consistent with others and just leave it under `tlsClient`.

Reply:
renamed in https://github.com/pingcap/tidb-operator/pull/3598/commits/bf5d600fe320f6b29377a0bee798abae3a87fc85